simgui-ds.json

```diff
@@ -96,8 +96,7 @@
   ],
   "robotJoysticks": [
     {
-      "guid": "78696e70757401000000000000000000",
-      "useGamepad": true
+      "guid": "Keyboard0"
```

Comment:
im seeing this through many PRs. consider changing this in main?

Reply:
see my note in carriage pr